### PR TITLE
Move EOF to the Osaka Fork

### PR DESCRIPTION
**Pull request by shemnon**
## PR description

* Ensure Osaka activations are working
* Remove PragueEOF fork
* Move EOF tools to default to Osaka

### Thanks for sending a pull request! Have you done the following?

- [ ] Checked out our [contribution guidelines](https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md)?
- [ ] Considered documentation and added the `doc-change-required` label to this PR [if updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).
- [ ] Considered the changelog and included an [update if required](https://wiki.hyperledger.org/display/BESU/Changelog).
- [ ] For database changes (e.g. KeyValueSegmentIdentifier) considered compatibility and performed forwards and backwards compatibility tests

### Locally, you can run these tests to catch failures early:

- [ ] unit tests: `./gradlew build`
- [ ] acceptance tests: `./gradlew acceptanceTest`
- [ ] integration tests: `./gradlew integrationTest`
- [ ] reference tests: `./gradlew ethereum:referenceTests:referenceTests`

### CODE REVIEW

File: evm/src/main/java/org/hyperledger/besu/evm/tracing/StandardJsonTracer.java [hunk 1]
```diff
@@ -127,9 +127,9 @@ public void tracePreExecution(final MessageFrame messageFrame) {
     for (int i = messageFrame.stackSize() - 1; i >= 0; i--) {
       stack.add("\"" + shortBytes(messageFrame.getStackItem(i)) + "\"");
     }
-    pc =
-        messageFrame.getPC()
-            - messageFrame.getCode().getCodeSection(messageFrame.getSection()).getEntryPoint();
+    pc = messageFrame.getPC() - messageFrame.getCode().getCodeSection(0).getEntryPoint();
+    //            -
+    // messageFrame.getCode().getCodeSection(messageFrame.getSection()).getEntryPoint();
```
Comment (fab-10) [inline evm/src/main/java/org/hyperledger/besu/evm/tracing/StandardJsonTracer.java:132]: remove?

File: evm/src/main/java/org/hyperledger/besu/evm/operation/CallFOperation.java [hunk 2]
```diff
@@ -30,6 +31,9 @@ public class CallFOperation extends AbstractOperation {
   /** The Call F success. */
   static final OperationResult callfSuccess = new OperationResult(5, null);
 
+  static final OperationResult callfStackOverflow =
```
Comment (fab-10) [inline evm/src/main/java/org/hyperledger/besu/evm/operation/CallFOperation.java:34]: this seems unrelated to the move to Osaka, is it intended to be here?

File: evm/src/main/java/org/hyperledger/besu/evm/tracing/StandardJsonTracer.java (hunk 1 above)
Comment (shemnon) [reply evm/src/main/java/org/hyperledger/besu/evm/tracing/StandardJsonTracer.java]: fixed

**Review by fab-10 — APPROVED** [review]
